### ISSUE TITLE
stop using my email for maintainer

### ISSUE THREAD

**Issue by xtraeme** [void-linux/void-packages#21303]
I resigned from my own project that I write from scratch since 2008.

I demand you to remove my email from maintainer's field in pkgs.

I'm not interested in continuing with this project that is full of idiots.

Have fun!

**Comment by Vaelatern** [comment]
It's meant "orphaned package" for years now. Nothing is broken.

**Comment by xtraeme** [comment]
Unless you remove my email I won't stop
moron

**Comment by xtraeme** [comment]
Yes, the same moron I added to my own organization that I resigned. How sarcastic!
fuck you idiot

**Comment by Vaelatern** [comment]
Please stop harassing me.

**Comment by xtraeme** [comment]
REMOVE MY FUCKING EMAIL ADDRESS
and update xbps to the REAL ADDRESS

**Comment by xtraeme** [comment]
I won't, you are a fucking moron

**Comment by xtraeme** [comment]
FUCK YOU MORON

**Comment by xtraeme** [comment]
you've been a bastard for you whole life, inst it?

ROFL
